### DISCUSSION TITLE
Do I need to merge commit: Implement PEP587 Python Initialzation Configuration.

### DISCUSSION THREAD

**Discussion by Alexey-T** [pyscripter/python4delphi#486] [Q&A] [answered]
@pyscripter 
I did not merge commit 
https://github.com/pyscripter/python4delphi/commit/9d4a980e3f6985a6d89fcfc03ef4b4898b098901
yet to Python4Lazarus.
but still, CudaText (based on python4lazarus) supports Python 3.13 on windows if I write the option "pylib": "python313.dll".

so commit is not required for 3.13 support for my app?

**Comment by pyscripter** [comment, marked as answer]
The reasons for this commit (see https://docs.python.org/3/whatsnew/3.13.html)

Remove the following old functions to configure the Python initialization, deprecated in Python 3.11:
- PySys_SetPath(): Set PyConfig.module_search_paths instead.
- Py_SetPath()
- _Py_SetProgramFullPath

**Pending Removal in Python 3.14  (originally scheduled for removal in 3.13) **
- PySys_SetArgvEx(): Set PyConfig.argv instead.
- PySys_SetArgv(): Set PyConfig.argv instead.
- Py_SetProgramName(): Set PyConfig.program_name instead.
- Py_SetPythonHome(): Set PyConfig.home instead.
- Py_DebugFlag: Use PyConfig.parser_debug instead.
- Py_VerboseFlag: Use PyConfig.verbose instead.
- Py_QuietFlag: Use PyConfig.quiet instead.
- Py_InteractiveFlag: Use PyConfig.interactive instead.
- Py_InspectFlag: Use PyConfig.inspect instead.
- Py_OptimizeFlag: Use PyConfig.optimization_level instead.
- Py_NoSiteFlag: Use PyConfig.site_import instead.
- Py_BytesWarningFlag: Use PyConfig.bytes_warning instead.
- Py_FrozenFlag: Use PyConfig.pathconfig_warnings instead.
- Py_IgnoreEnvironmentFlag: Use PyConfig.use_environment instead.
- Py_DontWriteBytecodeFlag: Use PyConfig.write_bytecode instead.
- Py_NoUserSiteDirectory: Use PyConfig.user_site_directory instead.
- Py_UnbufferedStdioFlag: Use PyConfig.buffered_stdio instead.
- Py_HashRandomizationFlag: Use PyConfig.use_hash_seed and PyConfig.hash_seed instead.
- Py_IsolatedFlag: Use PyConfig.isolated instead.

So sooner or later this commit will be needed. PySys_SetArgv(), Py_SetProgramName(), Py_SetPythonHome() are definitely needed.

**Comment by Alexey-T** [comment]
thanks. but for the work of Py 3.13, this commit is not needed, seems.
